scripts/dist.ts

```diff
@@ -246,7 +246,7 @@ function warnOnNeedlessOverrides(id, overridden, generated) {
 
 function main() {
   // Map from .yml to .dist.yml to filter out duplicates.
-  const sourceToDist = new Map(
+  const sourceToDist: Map<string, string> = new Map(
```

Comment:
If you prefer a little more brevity, this also works:

```suggestion
  const sourceToDist = new Map<string, string>(
```